src/world/area_kzn/kzn_03/kzn_03_2_npc.c

```diff
@@ -0,0 +1,740 @@
+#include "kzn_03.h"
+
+extern EvtScript D_800936DC;
+
+// spike top
```

Comment:
Perhaps these structs should be in an include?

Reply:
Creating enemy data includes and sorting that out will be a second pass once more maps are finished. Some considerable simplification is possible, but it will have to wait to ensure things are done in a way that works properly everywhere. As an example, Spike Tops share general include with Buzzy Beetles and others, along with 4 different AI configurations. So once all the maps that use them are sorted out, there can be a simple atomic include that just does the code+data for all of them.